src/config.rs

```diff
@@ -195,6 +200,26 @@ pub fn construct_config(pc: ProxyConfig) -> Result<Config, Error> {
         Some(parse_default(CA_ADDRESS, default_istiod_address)?)
     });
 
+    let xds_root_cert_provider = parse_default(
+        XDS_ROOT_CERT_PROVIDER_ENV,
+        DEFAULT_ROOT_CERT_PROVIDER.to_string(),
+    )?;
+    let xds_root_cert = if Path::new(&xds_root_cert_provider).exists() {
+        RootCert::File(xds_root_cert_provider.parse().unwrap())
```

Comment:
unwrap here isn't safe, can we propagate the error?

Reply:
It was fine before since it was for a constant

Reply:
sure, make sure no panic here.